pkgs/applications/version-management/git-and-tools/git/default.nix

```diff
@@ -65,7 +68,9 @@ stdenv.mkDerivation {
     ++ stdenv.lib.optionals perlSupport [ perl ]
     ++ stdenv.lib.optionals guiSupport [tcl tk]
     ++ stdenv.lib.optionals withpcre2 [ pcre2 ]
-    ++ stdenv.lib.optionals stdenv.isDarwin [ darwin.Security ];
+    ++ stdenv.lib.optionals stdenv.isDarwin [ darwin.Security ]
+    ++ stdenv.lib.optionals withLibsecret [ pkgconfig glib libsecret ]
+    ++ stdenv.lib.optionals withGnomeKeyring [ pkgconfig glib libgnome-keyring ];
```

Comment:
`libgnome-keyring` is deprecated in favour of `libsecret`, we do not want to use it.

Reply:
aha, good to know.  I was having trouble figuring out which one of those was deprecated in favor of the other; I'll drop the gnome-keyring version.